package customexceptions;

public class LoginExceptions {

	public static void main(String[] args) throws Exception {
		// TODO Auto-generated method stub

			ValidateUser.Validate(args[0], args[1]);

	}

}
